Formula/racer.rb

```diff
@@ -0,0 +1,34 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on "rust" => :build # rust MUST be up to date or else racer will produce invalid completions
+
+  resource "rust_source" do
+    url "https://static.rust-lang.org/dist/rustc-1.9.0-src.tar.gz"
```

Comment:
I think we could probably do something hacky where we make use of the existing Rust formula's tarball instead until we've got at least 2-3 formulae that need the source.